samples/batch/batch-ingestion-client/RealtimeTranscription/Helpers/ResultConversionHelper.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="ResultConversionHelper.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace RealtimeTranscription
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Xml;
+    using Connector;
+    using Microsoft.Extensions.Logging;
+
+    public static class ResultConversionHelper
+    {
+        public static SpeechTranscript CreateBatchResultFromRealtimeResults(string sourceName, List<JsonResult> realtimeResults, ILogger logger)
+        {
+            var timestamp = DateTime.UtcNow.ToString("O", CultureInfo.InvariantCulture);
+            var durationInTicks = realtimeResults.Select(jsonResult => jsonResult.Offset + jsonResult.Duration).Max();
+            var duration = XmlConvert.ToString(TimeSpan.FromTicks(durationInTicks));
+
+            logger.LogInformation("Converting combined recognized phrases");
+            var combinedRecognizedPhrases = realtimeResults
+                .Where(r => r.NBest.Any() && !string.IsNullOrEmpty(r.NBest.First().Lexical))
+                .GroupBy(r => r.SpeakerId, p => p, (key, g) => new
```

Comment:
does realtime transcription support SpeakerId? At least batch uses different diarization model than online, which might create different results.

Reply:
SpeakerId is only used here for multi channel audio, so Speaker = 0 will be channel = 0. I don't think we can perform Diarization for Realtime, or?